force-app/main/default/objects/HOT_HistoricallyAssignedResource__c/fields/IsDeleteStatusComment__c.field-meta.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>IsDeleteStatusComment__c</fullName>
+    <externalId>false</externalId>
+    <formula>ISPICKVAL(ServiceAppointment__r.Status, &quot;Completed&quot;)</formula>
```

Comment:
Skal dette også gjelde for Udekkede, Avlyste, Annullerte oppdrag?
Avklar med Svenn Terje

Reply:
Baser feltet på EarliestStartTime (at det skal være ett år siden)